system/src/Grav/Framework/Form/Traits/FormTrait.php

```diff
@@ -408,6 +409,7 @@ public function getAllFlashes(): array
     /**
      * {@inheritdoc}
      * @see FormInterface::render()
+     * @suppress PhanAccessMethodInternal - Twig render() is marked as internal
```

Comment:
This is not internal but our own implementation which has nothing to do with Twig.